### ISSUE TITLE
Philadelphia IPFS Meetup

### ISSUE THREAD

**Issue by flyingzumwalt** [ipfs/community#220]
Let's start a regular Philadelphia meetup. We could do it as a more presentation-based meetup or an IPFS study group covering some of the info discussed [here](https://github.com/ipfs/community/issues/186#issuecomment-254314223)

cc @gsf @bibliotechy @chadoh

**Comment by tjayrush** [comment]
I'm totally in. Very interested in IPFS.

**Comment by gsf** [comment]
Ideally we would balance doing real work (building apps and improving IPFS tools) with welcoming and educating people new to IPFS.

**Comment by chadoh** [comment]
I'm in! I'd especially be interested in a workshop to get some practical experience with deploying a simple front-end-only app to IPFS. Not currently capable of leading this myself.

**Comment by flyingzumwalt** [comment]
Our first Philadelphia IPFS Meetup will be in mid-April. Someone offered to host the meetup at their house in West Philly. I created a meetup.com group: https://www.meetup.com/Philadelphia-IPFS-Meetup

**Comment by tjayrush** [comment]
We're having an Ethereum meetup tomorrow if you guys are around: http://meetup.com/Philadelphia-Ethereum-Meetup.

**Comment by chadoh** [comment]
What day in mid-April, @flyingzumwalt? Other meetups and such are being planned, and I want to make sure I make it to this one.

Wish I could make it to the Ethereum meetup tonight, @tjayrush!

**Comment by flyingzumwalt** [comment]
@tjayrush sorry I can't make it. I'm out of town for a conference. 

We're thinking of either Monday April 17 or Tuesday April 18. Does anyone have a preference?

**Comment by gsf** [comment]
That Monday is better for me but either works. Go West Philly! Thanks for setting up the group, @flyingzumwalt.

Also hoping to make it to the ethereum meetup tonight.

**Comment by chadoh** [comment]
I will be speaking at the Philly react meet up that Wednesday, the 19th,
about offline first and distributed web stuff. I might be able to drum up
more attendance for this meet up, if we want to do it after the 19th.

**Comment by flyingzumwalt** [comment]
I'll be out of town for the following 2 weeks. Better to do it on the 17th and then have another meetup in May.

**Comment by tjayrush** [comment]
Either 17th or 18th works for me. We had a good turnout tonight--about 45.

**Comment by chadoh** [comment]
This is next week. Which day we going with? Let's get it on the calendar.

**Comment by flyingzumwalt** [comment]
Yes! Thank you for reminding. Let's do Tuesday April 18th. (I forgot that I'll be [teaching a meditation class](https://www.cultureworksphila.org/events/2017/4/6/mindful-mondays-at-cultureworks) on Monday)

**Comment by chadoh** [comment]
Perfect for me

**Comment by flyingzumwalt** [comment]
@gsf do you want to moderate? I put a little gesture of a schedule into the [event listing on meetup.com](https://www.meetup.com/Philadelphia-IPFS-Meetup/events/239142784/) but I like your suggestion that the format should be a mix of personal introductions, presentation, and working on stuff together.

**Comment by gsf** [comment]
I'm happy to moderate. Do we have a location?

**Comment by tjayrush** [comment]
I have a nice location at the WeWork offices in Northern Liberties if that works. How many people might attend? I would need a couple weeks notice prior to the date to make the reservation.

**Comment by flyingzumwalt** [comment]
We do have a location lined up for next week. Liz, whose last name I don't remember, is going to host at her house in West Philly. I'm just waiting to get the address so I can post it on the Meetup. @tjayrush thanks for offering to host at WeWork. I suspect that future meetups will be bigger and we will need that space for it, but this first meetup should be less than 30 people and it will be nice to have an informal setting for the first gathering.

**Comment by flyingzumwalt** [comment]
I got the address from liz. It's on the meetup event now. https://www.meetup.com/Philadelphia-IPFS-Meetup/events/239142784/

**Comment by diasdavid** [comment]
@flyingzumwalt please check https://github.com/ipfs/community/issues/283 when you have a moment :)
